packages/nimble-components/src/patterns/tab/styles.ts

```diff
@@ -58,7 +58,7 @@ export const styles = css`
         background: none;
     }
 
-    slot {
+    slot:not([name]) {
```

Comment:
This styling was meant to apply to just the default slot, which is all the `nimble-tab` provides*, but the `nimble-anchor-tab` has `start` and `end` slots, too. We don't want to pad them, or the tab becomes wider than we expect.

*I'm not sure _why_ the `nimble-tab` doesn't also have the `start` and `end` slots. In the FAST repo, the template we seemingly use has them.

Reply:
> I'm not sure why the nimble-tab doesn't also have the start and end slots. In the FAST repo, the template we seemingly use has them.

No sure what you mean, as mentioned above the tabs-toolbar leverages the end slot in the template automatically. Do you mean they are not styled to be useful, specifically the start slot? Then yea like the other spot linked to we should `display: none` unused / tested / designed slots.

Reply:
The tabs-toolbar uses the end slot of `nimble-tabs`, but I'm talking about the slots in individual tabs (i.e. `nimble-tab`). The [fast template](https://github.com/microsoft/fast/blob/21c210f2164c5cf285cade1a328460c67e4b97e6/packages/web-components/fast-foundation/src/tab/tab.template.ts#L14) has start and end slots, but when I examine the `nimble-tab` shadow DOM, I don't see them.

Based on this, I've decided to just remove the start and end slots from the `nimble-anchor-tab` template. I think it's still worth leaving my change to the CSS, just in case we add new slots in the future.